src/ykfde-open

```diff
@@ -16,11 +16,15 @@ while getopts ":d:n:hv" opt; do
 	v)	DBG=1
 		echo "debugging enabled"
 		;;
+	p)  YKFDE_PRINT_ONLY=1
```

Comment:
A little nitpick: can you fix indentation here to match the rest of options? :smile: